src/generator.cs

```diff
@@ -7081,5 +7103,12 @@ string RenderType (Type t)
 			return t.FullName;
 		
 	}
-	
+
+	public static string Quote (string s)
+	{
+		if (String.IsNullOrEmpty (s))
+			return s ?? String.Empty;
```

Comment:
🤔 Yeah this was intended mmm... But makes sense if I call Quote I expect a quoted anything